contributors/design-proposals/cpu-manager.md

```diff
@@ -158,6 +163,26 @@ node. Once allocated at pod admission time, an exclusive CPU remains
 assigned to a single container for the lifetime of the pod (until it
 becomes terminal.)
 
+##### Configuration
+
+Operators can set the number of CPUs that pods may run on through a new
+Kubelet configuration setting `--cpu-manager-static-num-cpus`, which
```

Comment:
Rewrite available for review: https://github.com/ConnorDoyle/community/pull/1/files